src/API/dashboard/dashboard.routes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import dashboardCtrl from './dashboard.controller.js';
+
+const router = express.Router();    // eslint-disable-line new-cap
+
+router.route('/')
+    /** GET /api/dashboard - Get dashboard */
+    .get(dashboardCtrl.query)
+
+    /** POST /api/dashboard - Create new dashboard */
+    .post(dashboardCtrl.create);
+
+router.route('/:id')
```

Comment:
!resolve
that I'm not quite getting: do we have ids for dashboard? can we add/delete dashboards?
dashboard itself for in my opinion shouldn't be RESTful (unless its settings for it, and still its debatable)
please, explain the flow with dashboard